azavea/raster-vision
# Document macOS setup instructions

## Overview

This PR updates the Raster Vision docs to include macOS specific setup instructions which I used to circumvent the problems I encountered in #487.

The minimal set of manually installation requirements seems to be:

```
pip install cython
pip install git+https://github.com/jswhit/pyproj.git
```

...before `pip install rastervision`. The git+https install for pyproj seems to be required on macOS, according to https://github.com/jswhit/pyproj/issues/138 and https://github.com/jswhit/pyproj/issues/136 

## Testing Instructions

From a clean install on macOS High Sierra, follow the the new macOS specific setup instructions and verify that they work.

After installation, you should be able to run

```
$ python3
>>> import rastervision
```

...without error.

Closes #487
